build.sbt

```diff
@@ -30,6 +30,8 @@ libraryDependencies ++= {
     "org.monifu"                  %% "monifu"                               % "1.0-RC3",
     "org.slf4j"                    % "slf4j-api"                            % "1.7.7",
     "ch.qos.logback"               % "logback-classic"                      % "1.0.13",
+    "org.scalaz"                  %% "scalaz-core"                          % "7.2.1",
```

Comment:
We should strive to add as few dependencies as possible. Also make contributions as easy as possible.

ScalaZ is a library recommended to be used after more than 6 months of experience with scala, I wouldn't include it here.

Beard is defined as a "lightweight template engine"... it should be lightweight in terms of dependencies also.